R/estimate_cumtime.R

```diff
@@ -0,0 +1,37 @@
+
+#' @title Estimate cumtime based on global speed
+#'
+#' @description Estimate cumtime of NA values of a GPS data based on the average
+#' speed of the valid GPS rows.
```

Comment:
suggestion for the documentation

```
#' @title Adjust the speeds of a gps-like table created with \code{\link{gtfs2gps}} 
#'
#' @description Some GTFS.zip data sets might have quality issues, for example 
#'              by assuming that a trip speed is unreasonably high (e.g. an urban
#'              bus running over 100 Km/h), or in other cases the `departure_time`
#'              information might be missing for some route segments. This can
#'              lead a gps-like table to have `NA` or unrealistic `speed` and 
#'              `departure_time` values. This function allows the user to ajust
#'              the speed of trips and updates `departure_time` values 
#'              accordingly. The user can ajust the problematic speeds by either
#'              setting a custom constant value, or by considering the average  
#'              of all valid trips speed (Default).
```